courses/free-courses-hi.md

```diff
@@ -270,6 +270,7 @@
 #### Node.js
 
 * [NodeJS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLsyeobzWxl7occsESx2X1E2R2Uw5wCoeG) - Telusko
+* [NodeJS Tutorial in Hindi](https://www.youtube.com/playlist?list=PL8p2I9GklV456iofeMKReMTvWLr7Ki9At) - Code Step By Step
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
I have corrected the title now. Please check.